### ISSUE TITLE
Log level can be configurable?

### ISSUE THREAD

**Issue by francesco-monorchio** [micronaut-projects/micronaut-test-resources#177]
### Feature description

When I run my application I see a lot of log from the test-resource server. Can be made configurable through the Gradle DSL?
Thanks.

**Comment by melix** [comment]
What logs do you see in particular? There shouldn't be a lot: mostly you should see containers starting.

**Comment by francesco-monorchio** [comment]
Yes, most of them are container startup logs but I have too many containers. Not a big deal but could be a good improvement IMHO. :-)

**Comment by alvarosanchez** [comment]
This is configurable today through the Gradle DSL by passing system properties to the test-resources server JVM, for example:

```gradle
micronaut {
    testResources {
        serverSystemProperties.put("logback.configurationFile", "logback-testresources.xml")
    }
}
```

Then define the logger levels you want in `logback-testresources.xml`, such as `org.testcontainers`, `org.testcontainers.dockerclient.DockerClientProviderStrategy`, or `io.micronaut.testresources`.

The key detail is that Micronaut Test Resources runs its server in a separate JVM, so your application's normal `logback.xml` does not directly control that process.

Because the configuration path already exists, this is being closed as a question rather than kept open as an implementation request.

---
###### ✨ This message was AI-generated using gpt-5.4
